benchmarking/evaluations/scenarios/concentrate_all_attributes.py

```diff
@@ -0,0 +1,30 @@
+from typing import List
+
+from models.enums import DiversifyType, ScenarioAttribute
+from benchmarking.evaluations.interfaces import Scenario, Goal
+from benchmarking.evaluations.goals import (
+    DiversityGoal,
+    WeightGoal,
+)
+
+
+class ConcentrateAllAttributes(Scenario):
```

Comment:
This is fine and described properly but as a nit; we'll need to chew on this name later maybe? Just since "all" implies we never add any others, which may or may not be true

It's good for now though and I think you specifying the included ones in the name makes it explicit

Reply:
I agree, but can't think of a better name that is descriptive, but not a crime against the eyes of anyone who reads it